File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let j=0
        nums.sort((a,b)=>a-b)
        for(let i=1;i<nums.length;i++){
            if(nums[i]!==nums[j]){
                nums[++j]=nums[i]
            }else{return true}
        }
        return false
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        HashMap<Integer,Integer>map=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            if(map.containsKey(nums[i])){
                int val=map.get(nums[i]);
                map.put(nums[i],val+1);
            }else{
                map.put(nums[i],1);
            }
        }
        ArrayList<Integer>key=new ArrayList<>(map.keySet());
        key.sort((a,b)->map.get(b)-map.get(a));
        int[] result=new int[k];
        for(int i=0;i<k;i++){
            result[i] = key.get(i);

        }
        return result;
    }
}
